### PR TITLE
Python Constraint Demo

### PR DESCRIPTION
**Pull request by alexandraj777**
Coming soon: R and Java!

### CODE REVIEW

File: constraints/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[![image](https://sigopt.com/static/img/SigOpt_logo_horiz.png?raw=true)](https://sigopt.com)
+
+# SigOpt Constraints
+
+Constraints can be added to a experiment definition to search for the optimum within a limited region. Once the constraints are defined, all the suggestions generated by SigOpt are guaranteed to satisfy the constraints.
+
+We have written examples using serveral different languages to demonstrate defining and using a SigOpt experiment with constraints.
```
Comment (benhsu75) [inline constraints/README.md]: typo

Comment (benhsu75) [inline constraints/README.md]: typo

Comment (benhsu75) [inline constraints/README.md]: using --> running?

Comment (alexandraj777) [reply constraints/README.md]: ?

Comment (alexandraj777) [reply constraints/README.md]: Found them!

**Pull request merged**